packages/api/tests/endpoints/test-granules.js

```diff
@@ -3899,3 +3903,29 @@ test.serial('default paginates correctly with search_after', async (t) => {
   t.not(results[0].granuleId, newResults[0].granuleId);
   t.not(meta.searchContext === newMeta.searchContext);
 });
+
+test.serial('PUT returns 400 for version value less than the configured value', async (t) => {
```

Comment:
Can you also test the case where the header is not set?

Reply:
Yes, although that reverse case is *technically* satisfied by all of the PUT functional tests.  That said it did highlight 🐜  assumptions in this test -- 142c052e4f